src/gridstack.js

```diff
@@ -1496,8 +1496,8 @@
         var relativeLeft = position.left - containerPos.left;
         var relativeTop = position.top - containerPos.top;
 
-        var columnWidth = Math.floor(this.container.width() / this.opts.width);
-        var rowHeight = Math.floor(this.container.height() / parseInt(this.container.attr('data-gs-current-height')));
+        var columnWidth = this.container.width() / this.opts.width;
+        var rowHeight = this.opts.cellHeight + this.opts.verticalMargin;
```

Comment:
here again, rounding too early causes problems with the calculations on longer grids